docs/resources/app.md

```diff
@@ -18,7 +18,8 @@ resource "ably_app" "app0" {
   status   = "enabled"
   tls_only = true
 }
-
+```
```

Comment:
Should there be a blank line between these two sets of backticks?

Reply:
Fixed in a8102db37f39001c5a161a00f7d1347e2080ec1f